app/controllers/users_controller.rb

```diff
@@ -1,58 +1,69 @@
-class UsersController < ApplicationController
-  before_action :set_user, only: [:show, :edit, :update, :destroy]
-
-  # GET /users
-  def index
-    @users = User.all
-  end
-
-  # GET /users/1
-  def show
-  end
-
-  # GET /users/new
-  def new
-    @user = User.new
-  end
-
-  # GET /users/1/edit
-  def edit
-  end
-
-  # POST /users
-  def create
-    @user = User.new(user_params)
-
-    if @user.save
-      redirect_to @user, notice: 'User was successfully created.'
-    else
-      render :new
-    end
-  end
-
-  # PATCH/PUT /users/1
-  def update
-    if @user.update(user_params)
-      redirect_to @user, notice: 'User was successfully updated.'
-    else
-      render :edit
-    end
-  end
-
-  # DELETE /users/1
-  def destroy
-    @user.destroy
-    redirect_to users_url, notice: 'User was successfully destroyed.'
-  end
-
-  private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_user
-      @user = User.find(params[:id])
-    end
-
-    # Only allow a trusted parameter "white list" through.
-    def user_params
-      params.require(:user).permit(:name, :admin)
-    end
-end
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  before_action :user_check, only:[:show, :edit, :destroy]
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @search_params = task_search_params
+    @tasks = Task.search(@search_params).order(params[:sort]).page(params[:page]).per(5)
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def edit
+  end

```

Comment:
このケースは1行で記載した方がベターですね。
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyMethod
```suggestion
  def edit; end
```

Reply:
ご指摘ありがとうございます。今後空白、省略されているメソッドは一行にまとめるようにいたします。